_opensearch/install/docker.md

```diff
@@ -322,3 +322,59 @@ In this case, `opensearch.yml` is a "vanilla" version of the file with no plugin
 cluster.name: "docker-cluster"
 network.host: 0.0.0.0
 ```
+
+## Sample Docker Compose file for development
+
+This sample file starts one OpenSearch node and a container for OpenSearch Dashboards for development by disabling security plugin.
+
```

Comment:
Minor rewording "You can use this sample file as a development environment. 
This sample file starts one OpenSearch node and a container for OpenSearch Dashboards with the security plugin disabled."

Reply:
Thanks, it's done.